website/api/zkvm/quickstart.md

```diff
@@ -5,93 +5,70 @@ slug: ./quickstart
 
 # zkVM Quick Start
 
-Welcome to the [zkVM] Quick Start page! Here are the steps between you and your first proof:
+Welcome to the zkVM Quick Start page! Here are the steps to create your first proof:
 
-1. Install the RISC Zero toolchain.
-2. Initialize a new project using the [cargo risczero] tool.
-3. Edit the boilerplate to build your first project.
-4. Run your project locally or with [Bonsai].
+1. Install the `cargo-risczero` toolchain
+2. Create a new project using the `cargo-risczero` tool.
+3. Familiarize yourself with the project's structure
+4. Run your project locally
```

Comment:
Can we align the outline and the list below? This overview shows 4 steps but then we walk through a 5 step process